packages/teleport/src/config.ts

```diff
@@ -114,6 +114,10 @@ const cfg = {
     nodeTokenPath: '/v1/enterprise/nodes/token',
     nodeScriptPath: '/scripts/:token/install-node.sh',
     appNodeScriptPath: '/scripts/:token/install-app.sh?name=:name&uri=:uri',
+
+    mfaSignRequestWithTokenPath: '/v1/webapi/u2f/signrequest/token',
+    mfaFetchDevicesPath: '/v1/webapi/mfadevice/:token',
+    mfaDeleteDevicePath: '/v1/webapi/mfadevice',
```

Comment:
```suggestion
    mfaDevicePath: '/v1/webapi/mfadevice',
```

if it was something like `/v1/webapi/mfadevice/delete` then `mfaDeleteDevicePath` would be better